zhaquirks/smarthjemmet/quadzigsw.py

```diff
@@ -0,0 +1,94 @@
+"""Device handler for smarthjemmet.dk QUAD-ZIG-SW."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.zcl.clusters.general import MultistateInput
+import zigpy.zcl.foundation
+
+from zhaquirks import PowerConfigurationCluster
+from zhaquirks.const import (
+    COMMAND,
+    COMMAND_DOUBLE,
+    COMMAND_HOLD,
+    COMMAND_RELEASE,
+    COMMAND_SINGLE,
+    COMMAND_TRIPLE,
+    DOUBLE_PRESS,
+    ENDPOINT_ID,
+    LONG_PRESS,
+    LONG_RELEASE,
+    SHORT_PRESS,
+    TRIPLE_PRESS,
+    VALUE,
+    ZHA_SEND_EVENT,
+)
+
+MANUFACTURER = "smarthjemmet.dk"
+MODEL_ID = "QUAD-ZIG-SW"
+
+ACTION_TYPE = {
+    0: COMMAND_RELEASE,
+    1: COMMAND_SINGLE,
+    2: COMMAND_DOUBLE,
+    3: COMMAND_TRIPLE,
+    4: COMMAND_HOLD,
+}
+
+
+class CR2032PowerConfigurationCluster(PowerConfigurationCluster):
+    """CR2032 Power Configuration Cluster."""
+
+    MIN_VOLTS = 2.2
+    MAX_VOLTS = 3.0
+
+
+class CustomMultistateInputCluster(CustomCluster, MultistateInput):
+    """Multistate input cluster."""
+
+    async def bind(self):
+        """Prevent bind."""
+        return zigpy.zcl.foundation.Status.SUCCESS
+
+    async def unbind(self):
+        """Prevent unbind."""
+        return zigpy.zcl.foundation.Status.SUCCESS
+
+    async def _configure_reporting(self, *args, **kwargs):
+        """Prevent remote configure reporting."""
+        return (
+            zigpy.zcl.foundation.ConfigureReportingResponse.deserialize(b"\x00")[0],
+        )
```

Comment:
Unrelated to converting the whole quirk to a v2 quirk:
Are these necessary? We already set `SKIP_CONFIGURATION = True`.